### ISSUE TITLE
Workflow item approved status indicator should be green, not white 

### ISSUE THREAD

**Issue by charliestrafe** [engram-design/Workflow#157]
### Describe the bug

I have an issue where the status indicator in the workflow overview table should be green, and not white. In the filter bar at the top of the workflow overview area it says 'Approved' with a green dot indicator, however when you actually approve and publish items they do not follow this colouring convention, they instead go to a white colour.
![Screenshot 2022-07-14 at 10 23 16](https://user-images.githubusercontent.com/47243887/178953251-24a92bc0-5f58-4c64-b1e0-5af63b21d2eb.png)

### Steps to reproduce

1. Submit item for review
2. Approve item
3. Publish item
4. Look in workflow overview area the is white, it should be green

### Craft CMS version

Craft Pro 4.1.4.1

### Plugin version

dev-craft-4 as 2.0.0-beta.4

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/workflow` requirement in `composer.json` to:

```
"require": {
  "verbb/workflow": "dev-craft-4 as 2.0.0-beta.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.5](https://github.com/verbb/workflow/releases/tag/2.0.0-beta.5)
